script_library/deployment-actions-openstack.sh

```diff
@@ -63,7 +63,6 @@ function clean_openstack(){
     run_ansible ${socok8s_absolute_dir}/playbooks/openstack-osh_instance.yml -e osh_node_delete=True
     echo "Delete CaaSP 3 nodes"
     run_ansible ${socok8s_absolute_dir}/playbooks/openstack-delete_caasp.yml
-    clean_caasp4
```

Comment:
so how do people remove caasp v4 if this line is deleted?

Reply:
by manually running `run.sh clean_caasp4`, which makes sense since deploying caasp4 requires a manual `run.sh deploy_caasp4`.